clay.h

```diff
@@ -28,41 +28,47 @@
 #define CLAY_WASM_EXPORT(null)
 #endif
 
+#ifdef _MSC_VER
+#define CLAY_PACKED_ENUM
```

Comment:
You make this `: uint8_t` instead of a noop in C++

Reply:
Nice idea, have put it in 👍